src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,143 @@ JSON example:
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_asl.json
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_aslcontext.tsv
+       [sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_m0scan.nii[.gz]]
+       [sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_m0scan.json]
+       [sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_labeling.jpg]
+```
+
+The complete ASL timeseries should be stored as a 4D NIfTI file in the original acquisition order, accompanied by two ancillary 
+files: `sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_asl.json` and `sub-<label>[_ses-<label>][_acq-<label>]
+[_rec-<label>][_run-<index>]_aslcontext.tsv`. The raw BIDS NIfTIs should contain appropriately scaled data and no scaling factors 
+should be stored in the ancillary files or in the NIfTI header. 
+
+The `m0scan` can either be stored inside the 4D ASL time-series NIfTI file or as a separate NIfTI file, depending on whether it 
+was acquired within the ASL time-series or as a separate scan. These and other M0 options are specified in the required M0 field. 
+It can also be stored under `fmap/sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_m0scan.nii[.gz]`, 
+when the pepolar approach is used.
+
+The `sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_aslcontext.tsv` lists the volume_type for each volume in 
+the `sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_asl.nii[.gz]`, corresponding to the volume_types 
+specified in the table below, based on DICOM Tag (0018,9257) `ASL Context`. Note that the volume_types `control` and 
+`label` within BIDS do not discriminate between types of labeling, but only serve to specify the magnetization state of the 
+blood and thus the ASL subtraction order.
+
+| volume_type                   | Definition    |
+|---------------------------------- | -----------------------------------------------------|
+| control                  |  The control image is acquired in the exact same way as the label image, except that the magnetization of the blood flowing into the imaging region has not been inverted.  |
+| label                  |The label image is acquired in the exact same way as the control image, except that the blood magnetization flowing into the imaging region has been inverted. |
+| m0scan                  | The M0 image is a calibration image, used to estimate the equilibruim magnetization of blood.        |
+| deltam                  | The deltaM image is a perfusion-weighted image, obtained by the subtraction of `control` - `label`.  |
+| cbf                  | The cerebral blood flow (CBF) image is produced by dividing the deltaM by the M0, quantified into `mL/100g/min` (See also <https://pubmed.ncbi.nlm.nih.gov/24715426/>).     |
```

Comment:
let's use DOIs wherever possible. My suggestion will render like this: [doi:10.1002/mrm.25197](https://doi.org/10.1002/mrm.25197).

Note that after accepting this suggestion, table pipes will have to be adjusted again.

```suggestion
| cbf                  | The cerebral blood flow (CBF) image is produced by dividing the deltaM by the M0, quantified into `mL/100g/min` (See also [doi:10.1002/mrm.25197](https://doi.org/10.1002/mrm.25197)).     |
```